app/assets/javascripts/goals.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```

Comment:
Unused `assets/*` files can be safely deleted to help keep the app clutter free.